lib/logsource.c

```diff
@@ -52,7 +52,10 @@ _flow_control_window_size_adjust(LogSource *self, guint32 window_size_increment)
   g_atomic_counter_set(&self->suspended_window_size, 0);
 
   if (old_window_size == 0)
-    log_source_wakeup(self);
+    {
+      msg_debug("Source has been resumed", evt_tag_str("group_name", self->options->group_name));
+      log_source_wakeup(self);
```

Comment:
Instead of the group name, either the location or the id should be logged. Also, wouldnt it make sense to add this log message to the resume function?